### ISSUE TITLE
Blocks List on Linux Unreadable

### ISSUE THREAD

**Issue by sylv256** [ClassiCube/MCGalaxy#651]
![image](https://user-images.githubusercontent.com/18603248/139957252-e9e1266a-9f01-4b4e-8f26-1ad3fbeae15b.png)
This is on Ubuntu 20.04 using Mono. The only way to make it readable is to use Wine.

**Comment by UnknownShadow200** [comment]
Which version of mono are you using? (`mono --version`)

**Comment by sylv256** [comment]
> Which version of mono are you using? (`mono --version`)

`Mono JIT compiler version 6.12.0.122`

**Comment by sylv256** [comment]
here's the logs
```
tehc@tiny-potato-pc:~$ '/home/tehc/MCGalaxy/MCGalaxy.exe' 
Gtk-Message: 18:03:25.724: Failed to load module "canberra-gtk-module"
SendMessage (56623186, 0xb5, 0x1, (nil))
SendMessage (56623186, 0xb5, 0x1, (nil))
SendMessage (56623186, 0xb5, 0x1, (nil))
SendMessage (56623186, 0xb5, 0x1, (nil))
SendMessage (56623186, 0xb5, 0x1, (nil))
SendMessage (56623186, 0xb5, 0x1, (nil))
SendMessage (56623186, 0xb5, 0x1, (nil))
SendMessage (56623186, 0xb5, 0x1, (nil))
SendMessage (56623186, 0xb5, 0x1, (nil))
SendMessage (56623186, 0xb5, 0x1, (nil))
SendMessage (56623186, 0xb5, 0x1, (nil))
SendMessage (56623186, 0xb5, 0x1, (nil))
```

**Comment by UnknownShadow200** [comment]
I assume that other list boxes in the GUI (Players tab, Settings -> Commands tab) show the same issue?

**Comment by sylv256** [comment]
> I assume that other list boxes in the GUI (Players tab, Settings -> Commands tab) show the same issue?

Yes

**Comment by UnknownShadow200** [comment]
What desktop environment and theme are you using?
![image](https://user-images.githubusercontent.com/6509348/140016240-f2293e46-e997-4d04-a6bb-38f8c6a8bfd3.png)

**Comment by forkiesassds** [comment]
Possibly a dark GTK theme.

**Comment by UnknownShadow200** [comment]
Issue happens with Yaru-Dark or Adwaita-Dark themes, will continue investigating to see if can fix

**Comment by UnknownShadow200** [comment]
Fixed this issue for all list boxes in 5b774651e36eb7973b989ba5485085c83bcfcb93

However property grids (e.g. map properties, level properties) still have this issue - tracking separately in #659
